### DISCUSSION TITLE
Number of days meal X was eaten

### DISCUSSION THREAD

**Discussion by sortab** [TandoorRecipes/recipes#1650] [Ideas]
Hi. Just started using Tandoor, so forgive my ignorance.

Is it possible to add a feature where one can see how long time ago a certain meal has been eaten? My wife is very fond of an app (that does not work any more) where she could sort by "time since last eaten" and see what we haven't had for dinner for a long time. It's a great way to rediscover some "forgotten" meals.

More specifically having this feature for both recipes and for the meal plan when no recipes are associated with a meal (e.g. when we have filled pitas where the kids can select their ingredients we don't use a recipe but it would be great to see that it's been X days since we last had pita).

**Comment by smilerz** [comment]
You can sort by “last cooked” as well as filter recipes by date list cooked and number of times cooked (both filters hidden by default)

**Reply by smilerz** [reply]
@sortab - in order to accomplish your use case for a Pita that doesn't really have a recipe you will need to create a recipe for that item.  Even if it is just a title without any other info.

**Reply by sortab** [reply]
Thank you very much for the help, exactly what I needed. Think I'll make some "empty" recipes for pita and other meals like that. Thanks again.

**Reply by sortab** [reply]
@smilerz Do you know if there is any way to "automatically" log a cooking of a recipe if the recipe has been planned in the meal planner? As far as I can see, I have to both add a recipe to the meal plan AND log it has been cooked in order to use the option to sort by "last cooked" / "no. times cooked".

**Reply by smilerz** [reply]
Not currently no. Good idea though, I recommend submitting a feature request.

**Reply by sortab** [reply]
OK. Just submitted a feature request in the Ideas categories.

**Reply by vabene1111** [reply]
i commented on how to probably best implement this. Should also think about merging comments into this things as well, could simplify some of the technical backend stuff and UI
